### PR TITLE
feat: User Profile page and file structure refactor

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
User can go to his and otheruser's profile page 
User can edit his profile info
File structure has been changed to group slices feature wise

### CODE REVIEW

**Review by AbhishekkGautam — APPROVED** [review]
Everything looks good. 
- clean code
- well structured folders.

File: src/features/UserProfile/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+					{allPosts?.map((post) => (
+						<Box
+							borderBottom={"1px solid"}
+							borderBottomColor={colorToggler(600)}
+						>
+							<Center>
+								<PostCard postDetails={post} />
+							</Center>
+							{/* <Divider orientation="horizontal" color={colorToggler(400)} /> */}
+						</Box>
```
Comment (gopalbbharadva) [inline src/features/UserProfile/UserProfile.jsx:61]: Remove these comments if not needed

File: src/features/Home/Home.jsx [hunk 2]
```diff
@@ -15,23 +21,13 @@ export const Home = () => {
 	useEffect(() => dispatch(getPosts()), []);
 	console.log(allPosts);
```
Comment (gopalbbharadva) [inline src/features/Home/Home.jsx]: Remove this console.

**Review by gopalbbharadva — APPROVED** [review]
Functionality working fine, Nice UI, Well - code structure,

**Pull request merged**